Pages/Index.cshtml

```diff
@@ -15,28 +15,64 @@
             <li>Search Brewery Names </li>
             <li>Search Brewery brewery_type</li>
         </ul>
-    </div>      
+    </div>
+
+    <form method="get" action="/Search" class="mb-4">
+        <div class="row">
+            <div class="col-md-6 mb-3">
+                <label for="state" class="form-label">State</label>
+                <input type="text"
+                       class="form-control"
+                       id="state"
+                       name="State"
+                       placeholder="Enter state name (e.g. California)">
+            </div>
+            <div class="col-md-6 mb-3">
+                <label for="breweryType" class="form-label">Brewery Type</label>
+                <select class="form-select" id="breweryType" name="BreweryType">
```

Comment:
Optimized Dropdown Logic: Refactored dropdown logic for concise conditions, following C# Coding Conventions (https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions), enhancing readability and maintainability.